src/platform/graphics/webgl/webgl-texture.js

```diff
@@ -512,10 +512,13 @@ class WebglTexture {
                         }
                     }
 
+                    const w = mipObject.width || mipObject.videoWidth;
```

Comment:
?? would probably be better

Reply:
For video textures width and height are 0.